addon/components/x-tree-branch.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class XTreeBranchComponent extends Component {
+  
+  @action
+   onLoad(modell){
```

Comment:
- modell to model.
- Can we directly use the onLoad function in the component, as we are not writing anything inside the method.

Reply:
I think we cant use it directly. I referred the addon code, they described other properties such as onSelect, onHover, onHoverout... like, 
` @action
  mouseEnter() {
    if (!get(this.args.model, 'isDisabled')) {
      set(this.args.model, 'isSelected', true);
    }


    if (this.args.onHover) {
      this.args.onHover(this.args.model);
    }
  }

  @action
  mouseLeave() {
    set(this.args.model, 'isSelected', false);

    if (this.args.onHoverOut) {
      this.args.onHoverOut(this.args.model);
    }
  }`